jenkins_cli/cli.py

```diff
@@ -31,6 +32,11 @@
                   }
 
 
+# Default constants to parse booleans from .jenkins-cli
+TRUE = ['yes', 'Yes', 'YES', 'True', 'true', 'TRUE', '1']
```

Comment:
I agree that ```ignore_ssl=False``` may looks a bit strange. But the list with all acceptable strings looks strange also.
Maybe it will be enough to define behavior in help section?

Something like:
 "You can set ignore_ssl in settings file. Any value will be treated as True unless it equals 0"
or
 "You can set ignore_ssl=1 in settings file if you want to ignore ssl erros. All other values will be treated as False"

We need to document this option anyway and its hardy possible to find all acceptable strings (for example we can also add 'y' and 'Y')

What do you think?